Extension/src/LanguageServer/client.ts

```diff
@@ -2976,6 +3035,55 @@ export class DefaultClient implements Client {
         }
     }
 
+    public async handleGenerateDoxygenComment(line: number | undefined, column: number | undefined): Promise<void> {
+        const editor: vscode.TextEditor | undefined = vscode.window.activeTextEditor;
+        if (!editor) {
+            return;
+        }
+        const params: GenerateDoxygenCommentParams = {
+            uri: editor.document.uri.toString(),
+            position: (line !== undefined && column !== undefined) ? new vscode.Position(line, column) : editor.selection.active,
+            isCodeAction: (line !== undefined && column !== undefined) ? true : false
+        };
+
+        if (line !== undefined && column !== undefined) {
```

Comment:
Can this check to determine the `params.position` and `params.isCodeAction` values be done during the initialization of `params`? If so, then this additional check and work can be removed.